appcli/cli_builder.py

```diff
@@ -125,6 +125,11 @@ def create_cli(configuration: Configuration, desired_environment: Dict[str, str]
         multiple=True,
         callback=extract_valid_environment_variable_names,
     )
+    @click.option(
+        "--non-tty",
```

Comment:
In Unix, the general standard for negative flags is `--no-X`. See `man git`, or `man grep` (search for `--no` flags), or the click documentation about flags https://click.palletsprojects.com/en/7.x/options/#boolean-flags

Given that, I think it would be better if we specify both the positive and negative cases, like in the click example, so we don't have the double negation:
```
@click.option('--tty/--no-tty, default=True)
```